crates/networking/rpc/eth/filter.rs

```diff
@@ -7,21 +7,51 @@ use std::{
 use ethereum_rust_storage::Store;
 use tracing::error;
 
-use crate::utils::{RpcErr, RpcRequest};
+use crate::utils::{parse_json_hex, RpcErr, RpcRequest};
 use crate::RpcHandler;
 use rand::prelude::*;
 use serde_json::{json, Value};
 
 use super::logs::LogsFilter;
 
 #[derive(Debug, Clone)]
-pub struct FilterRequest {
+pub struct NewFilterRequest {
     pub request_data: LogsFilter,
 }
 
+/// Used by the tokio runtime to clean outdated filters
+/// Takes 2 arguments:
+/// - filters: the filters to clean up.
+/// - filter_duration: represents how many *seconds* filter can last,
+///   if any filter is older than this, it will be removed.
+pub fn clean_outdated_filters(filters: ActiveFilters, filter_duration: u64) {
```

Comment:
If it isn't much of trouble, we could use the `Duration` type for `filter_duration` instead of being a `u64`

Reply:
Done!